diff --git a/chrome/browser/ui/page_info/page_info_ui.cc b/chrome/browser/ui/page_info/page_info_ui.cc
index 4eb66e247b4adde8c3600ac71c33cf42705e2043..2fd2923033f2cd797f66063fd40119bd4251d2ef 100644
--- a/chrome/browser/ui/page_info/page_info_ui.cc
+++ b/chrome/browser/ui/page_info/page_info_ui.cc
@@ -479,6 +479,10 @@ const gfx::ImageSkia PageInfoUI::GetPermissionIcon(const PermissionInfo& info,
     case CONTENT_SETTINGS_TYPE_USB_GUARD:
       icon = &vector_icons::kUsbIcon;
       break;
+    case CONTENT_SETTINGS_TYPE_AUTOPLAY:
+      // TODO(darkdh): autoplay icon
+      icon = &kExtensionIcon;
+      break;
     default:
       // All other |ContentSettingsType|s do not have icons on desktop or are
       // not shown in the Page Info bubble.
