diff --git a/chrome/browser/ui/webui/settings/metrics_reporting_handler.h b/chrome/browser/ui/webui/settings/metrics_reporting_handler.h
index 73cc5bd9572e3b49ed94697be0f48bb089347a70..d2e1a4a82eb5fff290f077fa0b5f9ba83e0fa9d2 100644
--- a/chrome/browser/ui/webui/settings/metrics_reporting_handler.h
+++ b/chrome/browser/ui/webui/settings/metrics_reporting_handler.h
@@ -5,7 +5,7 @@
 #ifndef CHROME_BROWSER_UI_WEBUI_SETTINGS_METRICS_REPORTING_HANDLER_H_
 #define CHROME_BROWSER_UI_WEBUI_SETTINGS_METRICS_REPORTING_HANDLER_H_
 
-#if defined(GOOGLE_CHROME_BUILD) && !defined(OS_CHROMEOS)
+#if (defined(GOOGLE_CHROME_BUILD) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))) && !defined(OS_CHROMEOS)
 
 #include <memory>
 
