diff --git a/components/crash/content/app/crashpad_win.cc b/components/crash/content/app/crashpad_win.cc
index 8b0edef1b59403f819e3c1b4d3b378f849c68f26..3db3b80b9d1f184714570aff5ec4b86cf6bc29fa 100644
--- a/components/crash/content/app/crashpad_win.cc
+++ b/components/crash/content/app/crashpad_win.cc
@@ -36,7 +36,7 @@ void GetPlatformCrashpadAnnotations(
       exe_file, &product_name, &version, &special_build, &channel_name);
   (*annotations)["prod"] = base::UTF16ToUTF8(product_name);
   (*annotations)["ver"] = base::UTF16ToUTF8(version);
-#if defined(GOOGLE_CHROME_BUILD)
+#if defined(GOOGLE_CHROME_BUILD) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))
   // Empty means stable.
   const bool allow_empty_channel = true;
 #else
@@ -82,6 +82,8 @@ base::FilePath PlatformCrashpadInitialization(bool initial_client,
 
 #if defined(GOOGLE_CHROME_BUILD)
     std::string url = "https://clients2.google.com/cr/report";
+#elif defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
+    std::string url = "https://laptop-updates.brave.com/1/bc-crashes";
 #else
     std::string url;
 #endif
