diff --git a/components/browser_sync/profile_sync_service.cc b/components/browser_sync/profile_sync_service.cc
index edb33dafe55b88f95ad99542260e12a1f97f30e2..6e0c49f94ab3892088c567bee974139ec2b7386e 100644
--- a/components/browser_sync/profile_sync_service.cc
+++ b/components/browser_sync/profile_sync_service.cc
@@ -138,7 +138,7 @@ ProfileSyncService::InitParams::InitParams(InitParams&& other) = default;
 ProfileSyncService::InitParams::~InitParams() = default;
 
 ProfileSyncService::ProfileSyncService(InitParams init_params)
-    : SyncServiceBase(std::move(init_params.sync_client),
+    : BraveSyncServiceBase(std::move(init_params.sync_client),
                       std::move(init_params.signin_wrapper),
                       init_params.channel,
                       init_params.base_directory,
