diff --git a/third_party/blink/renderer/core/html/media/autoplay_policy.h b/third_party/blink/renderer/core/html/media/autoplay_policy.h
index 315ab2dd43fa7eff4cdbd42dedec97726e2c2cd7..0e4593670ff470238ac913e3d52e0b94e2d5daa3 100644
--- a/third_party/blink/renderer/core/html/media/autoplay_policy.h
+++ b/third_party/blink/renderer/core/html/media/autoplay_policy.h
@@ -5,6 +5,7 @@
 #ifndef THIRD_PARTY_BLINK_RENDERER_CORE_HTML_MEDIA_AUTOPLAY_POLICY_H_
 #define THIRD_PARTY_BLINK_RENDERER_CORE_HTML_MEDIA_AUTOPLAY_POLICY_H_
 
+#include "third_party/blink/public/platform/modules/permissions/permission.mojom-blink.h"
 #include "third_party/blink/renderer/core/core_export.h"
 #include "third_party/blink/renderer/core/dom/exception_code.h"
 #include "third_party/blink/renderer/platform/heap/handle.h"
@@ -137,6 +138,8 @@ class AutoplayPolicy final : public GarbageCollected<AutoplayPolicy> {
   // frame.
   bool IsAutoplayAllowedPerSettings() const;
 
+  void HandleAutoplayWithPermission(mojom::blink::PermissionStatus status) const;
+
   bool IsAutoplayingMutedInternal(bool muted) const;
   bool IsOrWillBeAutoplayingMutedInternal(bool muted) const;
 
